src/markdown.rs

```diff
@@ -23,18 +23,24 @@ pub fn prepare_markdown(text: &str) -> String {
     return buf;
 }
 
-pub fn make_skin() -> termimad::MadSkin {
+fn make_skin() -> termimad::MadSkin {
```

Comment:
Let's store `MadSkin` in a `once_cell::sync::Lazy`. As even if `MadSkin` constructor is fast enough, we do a syscall in the function (`atty::is`). Doing that per each piece of text may not be very noticeable on fast desktop CPUs, but may be slow on Raspberry PIs or cheap VMs exhausted their burst capabilities.